.github/workflows/generatemd.yml

```diff
@@ -0,0 +1,66 @@
+name: Generate Proposal MD
+
+on:
+  workflow_dispatch: #periodic
+    inputs:
+      logLevel:
+        description: "Log lev"
+        required: true
+        default: "warning"
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  generateHTML:
+    env:
+      JBAL_VERSION: "2201.0.3"
+      GITHUB_TOKEN: ${{secrets.BALLERINA_BOT_TOKEN}}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: website-revamp
+          token: ${{ secrets.BALLERINA_BOT_TOKEN }}
+      - name: Run Proposals Bot
+        uses: ballerina-platform/ballerina-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+        with:
+          args: 
+            run .github/scripts/bal_proposals_bot
+      - name: Commit files and Push
+        id: commit
+        run: |
+          DIFF_VAR=false
+          if git diff --quiet HEAD --; then
+              echo "No changes"             
+          else
+              DIFF_VAR=true
+              UUID_VAL=$(uuidgen)
+              echo $UUID_VAL
```

Comment:
@bimsarapathiraja Do we need this echo? is it for testing purpose? If so lets remove this

Reply:
Removed